packages/components/stories/date-picker-native/stories/basic.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { boolean } from '@storybook/addon-knobs';
 import { toMoment } from '@deriv/shared';
-import DatePickerNative from 'Components/date-picker/date-picker-native';
+import DatePickerNative from '../../../src/components/date-picker/date-picker-native.jsx';
```

Comment:
do we have aliases for this?

Reply:
Hello @ali-hosseini-fs,

This problem actually exists in multiple packages. I thought this `eslint-import-resolver-webpack` package would resolve it, but it already exists in the project and they're still popping up. I'll try to look for another workaround for this
<img width="997" alt="import resolver" src="https://user-images.githubusercontent.com/104334373/178637410-6b45aef4-d9eb-4384-8a5f-148aa380c980.png">

Reply:
The solution is [here](https://github.com/import-js/eslint-plugin-import#resolvers)

Reply:
@jim-deriv So, can we add an alias in eslint config?

Reply:
Hello @balakrishna-binary,

I have included the fix in the latest changes. It needs to be done in the webpack config file